fpga_interchange/arch_pack_clusters.cc

```diff
@@ -370,6 +506,344 @@ static bool check_cluster_cells_compatibility(CellInfo *old_cell, CellInfo *new_
     return true;
 }
 
+bool reduce(uint32_t x, uint32_t y, const ClusterPOD *cluster, dict<uint32_t, pool<CellInfo *, hash_ptr_ops>> &domain, Context *ctx){
```

Comment:
this could do with a more descriptive name, or a comment describing what it's doing

Reply:
Added comment in new commit